tests/modules/nf-core/bbmap/bbnorm/main.nf

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { BBMAP_BBNORM } from '../../../../../modules/nf-core/bbmap/bbnorm/main.nf'
+
+workflow test_bbmap_bbnorm_se {
+    
+    input = [
+        [ id:'test', interleaved:true ], // meta map
+        file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true)
+    ]
+
+    BBMAP_BBNORM ( input )
+}
+
+workflow test_bbmap_bbnorm_pe {
+
+    input = [ 
+        [ id:'test', interleaved:false ], // meta map
+        [  
+            file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true),
+           file(params.test_data['sarscov2']['illumina']['test_2_fastq_gz'], checkIfExists: true) 
+       ]
```

Comment:
```suggestion
            file(params.test_data['sarscov2']['illumina']['test_2_fastq_gz'], checkIfExists: true) 
        ]
```
It's difficult to get this right as the GitHub editor uses a proportional font. Make sure indentation is by four spaces.